mungegithub/mungers/publish_scripts/clientgo_publish.sh

```diff
@@ -43,6 +44,23 @@ if git diff --cached --exit-code &>/dev/null; then
     exit 0
 fi
 git commit -m "${MESSAGE}"
+
+# Run "godep restore" to restore dependencies. Because entries for
```

Comment:
vendor/ in staging directory is fixed now, so i'm going to rewrite this part of code to simply replace the SHA of k8s.io/apimachinery in Godeps.json, but not update vendor/. Then the script doesn't need to go through the `godep save/restore`.